### PR TITLE
App startup bugs

### PR DESCRIPTION
**Pull request by SamTremko**

### CODE REVIEW

**Comment by SamTremko** [comment]
* **#860** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/860?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#859** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/859?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#830** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/830?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#829** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/829?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#828** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/828?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @SamTremko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/mobile/src/AnimatedSplashScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  useEffect(() => {
+    if (fontsLoaded && sessionLoaded && remoteConfigSetup) {
+      void SplashScreen.hideAsync().then(() => {
+        setIsAppReady(true)
+      })
+    }
+  }, [fontsLoaded, sessionLoaded, remoteConfigSetup])
+
+  return (
+    <Stack f={1}>
```
Comment (kaladivo) [inline apps/mobile/src/AnimatedSplashScreen.tsx:73]: Position='relative' should be added here right?

Comment (kaladivo) [inline apps/mobile/src/AnimatedSplashScreen.tsx:73]: Position='relative' should be added here right?

File: apps/mobile/src/App.tsx [hunk 2]
```diff
@@ -95,7 +58,9 @@ function App(): JSX.Element {
 export default function _(): JSX.Element {
   return (
     <ThemeProvider>
-      <App />
+      <AnimatedSplashScreen>
+        <App />
```
Comment (kaladivo) [inline apps/mobile/src/App.tsx:62]: Will this really prevent rendering the whole app until splashScreen is finished?

**Review by kaladivo — CHANGES_REQUESTED** [review]
Let's talk about this tmrw

**Review by kaladivo — APPROVED** [review]

**Comment by SamTremko** [comment]
### Merge activity

* **Mar 8, 5:20 AM EST**: @SamTremko merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/859).

**Pull request merged**